onnx_tf/handlers/backend/clip.py

```diff
@@ -1,26 +1,37 @@
 import tensorflow as tf
 
 from onnx_tf.common import exception
+from onnx_tf.common import sys_config
 from onnx_tf.handlers.backend_handler import BackendHandler
 from onnx_tf.handlers.handler import onnx_op
-from onnx_tf.handlers.handler import partial_support
-from onnx_tf.handlers.handler import ps_description
+import onnx_tf.common.data_type as data_type
 
 
 @onnx_op("Clip")
-@partial_support(True)
-@ps_description("Clip input in uint64 is not supported in Tensorflow.")
 class Clip(BackendHandler):
+  cast_map = {
+      tf.uint8: tf.int32,
+      tf.uint16: tf.int32,
+      tf.uint32: tf.int64,
+      tf.int8: tf.int32,
+      tf.int16: tf.int32
+  }
+  cast_map[tf.uint64] = tf.int64 if sys_config.auto_cast else None
```

Comment:
@chinhuang007 Please move line 19 into args_check. With current location, auto_cast will always be false and user can not change it at all.

Reply:
Good catch! Fixed, please review again